lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -298,7 +298,7 @@ PullImportPaths pullImport(
 
     std::string mFileName = context.GetUsdStage()->GetRootLayer()->GetIdentifier();
     if (mFileName.empty()) {
-        TF_RUNTIME_ERROR("Empty file specified. Exiting.");
+        TF_WARN("Empty file specified. Exiting.");
```

Comment:
"Exiting" wording feels misleading, and so does talking about a file --- we're pulling on USD data that's in memory.